plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeUpdatablePageSource.java

```diff
@@ -574,7 +575,7 @@ private ReaderPageSource createParquetPageSource(TupleDomain<HiveColumnHandle> p
                 fileSize,
                 columns,
                 parquetPredicate,
-                true,
+                NAME,
```

Comment:
Can you add a TODO comment that explains that this is safe because we don't have writes to tables with column mappings enabled yet?